codegen/src/test/resources/q3/Q3.json

```diff
@@ -0,0 +1,120 @@
+{
+  "join_structure": [
+    {
+      "PrimaryRelation": "Orders",
+      "ForeignRelation": "Lineitem"
+    },
+    {
+      "PrimaryRelation": "Orders",
+      "ForeignRelation": "Customer"
+    }
+  ],
+  "LineitemProcessFunction": {
```

Comment:
All Relation Process Functions should represents like the join structure with an array.